### PR TITLE
Adding manual installation notes

### PR DESCRIPTION
**Pull request by Jazz747**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,40 @@ Here's how to use it:
     ```js
     import Snackbar from 'react-native-snackbar';
     ```
+## Manual installation
+
+In case step 2 above throws errors:
```
Comment (iRoachie) [inline README.md]: You can remove this line

File: README.md [hunk 2]
```diff
@@ -61,6 +61,40 @@ Here's how to use it:
...
+
+2. insert the following lines of code:
```
Comment (iRoachie) [inline README.md]: Remove here too

File: README.md [hunk 3]
```diff
@@ -61,6 +61,40 @@ Here's how to use it:
...
+
+In `android/settings.gradle`:
+```
```
Comment (iRoachie) [inline README.md]: So we get syntax highlighting, add java after ```.

E.g. \```java

You only need to add it to the opening of the code blocks. So do it for three code blocks

**Review by iRoachie — COMMENTED** [review]
@Jazz747 After making these changes it should be good to go

**Review by iRoachie — APPROVED** [review]
Awesome work @Jazz747

**Comment by cooperka** [comment]
Hi @Jazz747, thanks for the PR! I'm currently taking a vacation but I'll be back around May 23 to review and merge this :)

**Review by cooperka — APPROVED** [review]
I moved your docs to the wiki and just added a link to the README. Thanks again for this!

**Pull request merged**